website/templates/project/nodes_delete.mako

```diff
@@ -84,6 +85,7 @@
                     </span>
```

Comment:
The Delete button here can be clicked before the confirmation word is typed.  My opinion, button should be disabled until strings match.
<img width="587" alt="screen shot 2018-01-17 at 2 12 42 pm" src="https://user-images.githubusercontent.com/9755598/35064767-bc3c7eea-fb90-11e7-96c4-5d96e6d84037.png">